cmd/validate.go

```diff
@@ -91,14 +95,17 @@ func (vc *validateCmd) run(c *cobra.Command) error {
 		return fmt.Errorf("not a valid file or directory: %w", err)
 	}
 
-	var manifestFiles []string
+	var manifestFiles []safeguards.ManifestFile
 	if isDir {
```

Comment:
is this isDir branch necessary? the description of walkdir and walkdirtree say it should run on a single file as well as a directory recursively
> walks the file tree rooted at root, calling fn for each file or directory in the tree, **including root** [link](https://pkg.go.dev/path/filepath#Walk)
this means we can likely just always call the code in the isDir case

Reply:
I will look into this in a future PR